app/views/users/choose_identity.html.erb

```diff
@@ -7,7 +7,7 @@
 	<%- if @current_user.is_a?(PubcookieUser) -%>
 		<li><%= link_to "Dream Project mentor", update_identity_path(:identity => "Mentor"), :method => :post %></li>
 	<%- else -%>
-		<li><%= link_to "high-school student", update_identity_path(:identity => "Student"), :method => :post %></li>
+		<li><%= link_to "high-school student", update_identity_path(:identity => "Participant"), :method => :post %></li>
```

Comment:
Oh. And then mentees can fill out the intake form on their own? That sounds a good plan.